src/common/SplitButton/splitButton.scss

```diff
@@ -24,6 +24,7 @@
     button {
       margin: 0;
       min-width: inherit;
+      min-height: 40px;
```

Comment:
splitButton should have the same height as the Button. Button has the height 36px. I don't think you should add `min-height` here